valueformatting_test.go

```diff
@@ -267,6 +267,30 @@ func TestValueFormattingBinaryFormatter(t *testing.T) {
 	}
 }
 
+func TestValueFormattingJSONFormatter(t *testing.T) {
+	formatting := newValueFormatting()
+	formatter, err := formatting.jsonFormatter()
+
+	if err != nil {
+		t.Errorf("Error creating formatter: %v", err)
+	}
+
+	input := []byte("{\"name\": \"Brave\", \"age\": 2}")
```

Comment:
Since the code under test has many more cases than just `string` and `int`, this should probably include those others cases, too.

Reply:
Good point. Done.